src/components/EndSession.jsx

```diff
@@ -104,16 +91,47 @@ const Button = styled.button`
 `;
 
 function EndSession() {
+    const emailRef = useRef();
+    const sessionCode = sessionStorage.getItem("sessionCode");
+
+    useEffect(() => emailjs.init("OScF2lHq5ESl_o9lU"), []);

```

Comment:
Are these api keys? If they are, they should be moved to the environment variables.

Reply:
Oh if it's a public api then it's fine. I thought it was private one.